README.md

```diff
@@ -1,2 +1,249 @@
-# FOOOF.jl
+# PyFOOOF.jl
 Julia interface to [FOOOF](https://github.com/fooof-tools/fooof)
+
+[![Build Status][build-img]][build-url] [![CodeCov][codecov-img]][codecov-url]
+
+[build-img]: https://github.com/beacon-biosignals/PyFOOOF.jl/workflows/CI/badge.svg
+[build-url]: https://github.com/beacon-biosignals/PyFOOOF.jl/actions
+[codecov-img]: https://codecov.io/github/beacon-biosignals/PyFOOOF.jl/badge.svg?branch=main
+[codecov-url]: https://codecov.io/github/beacon-biosignals/PyFOOOF.jl?branch=main
+
+
+## Installation
+This package uses [`PyCall`](https://github.com/JuliaPy/PyCall.jl/) to make
+[FOOOF](https://fooof-tools.github.io/fooof/index.html) available from within Julia.
+Unsurprisingly, FOOOF and its dependencies need to be installed in order for this to work
+and PyFOOOF will attempt to install when the package is built.
+
+By default, this installation happens in the "global" path for the Python used
+by PyCall. If you're using PyCall via its hidden Miniconda install, your own
+Anaconda environment, or a Python virtual environment, this is what you want.
+(The "global" path is sandboxed to the Conda/virtual environment.) If you are
+however using system Python, then you should set `ENV["PIPFLAGS"] = "--user"`
+before `add`ing / `build`ing the package. By default, PyFOOOF will use the latest
+FOOOF 1.x release available on [PyPI](https://pypi.org/project/FOOOF/), but this can also
+be changed via the `ENV["FOOOFVERSION"] = version_number` for your preferred
+`version_number`.
+
+Note that FOOOF-Python uses [`matplotlib`](https://matplotlib.org/) for plotting, but does not install it automatically as a dependency.
+If you wish to take advantage of this functionality, the non-exported `install_matplotlib` function will install `matplotlib`, using the same environment variables as the main installation.
+The Julia package [`PyPlot`](https://github.com/JuliaPy/PyPlot.jl), which provides a Julia interface to `matplotlib`, is also useful for manipulating the rendered plots.
+
+FOOOF-Python can also be installed manually ahead of time.
+From the shell, use `python -m pip install fooof` for the latest stable release
+or `python -m pip install fooof==version_number` for a given `version_number`,
+ensuring  that `python` is the same one that PyCall is using. Alternatively,
+you can run this from within Julia:
+```julia
+julia> using PyCall
```

Comment:
Inconsequential nitpick, but the fake REPL sessions in docs bother me (i.e. ones that don't show output so you know they're editied or falsified). I'd just remove the prompts.

Reply:
The prompts are here because there are also Python examples